gr-filter/include/gnuradio/filter/pfb_channelizer_ccf.h

```diff
@@ -77,12 +77,12 @@ namespace gr {
      * bandwidth of <EM>TB</EM>. We can also specify the out-of-band
      * attenuation to use, <EM>ATT</EM>, and the filter window
      * function (a Blackman-harris window in this case). The first input
-     *  is the gain of the filter, which we specify here as unity.
+     * is the gain of the filter, which we specify here as unity.
      *
-     *      <B><EM>self._taps = filter.firdes.low_pass_2(1, fs, BW, TB,
-     *           attenuation_dB=ATT, window=filter.firdes.WIN_BLACKMAN_hARRIS)</EM></B>
+     *    <B><EM>self._taps = filter.firdes.low_pass_2(1, fs, BW, TB,
+     *       attenuation_dB=ATT, window=filter.firdes.WIN_BLACKMAN_hARRIS)</EM></B>
```

Comment:
This is not relevant to your changes, but it's odd that the 'h' is not capitalized.

Reply:
I presume it's that way because harris prefers his name in lower case. :smile: https://en.wikipedia.org/wiki/Fredric_J._Harris

I see an alias has been available for some time: https://github.com/gnuradio/gnuradio/commit/f0c9a22db5442ff6029a0561bb2178de851c6b77

Reply:
That's beautiful. I knew in the back of my mind about his preference, but would not have thought to have inverted it for the comment.